extensions/wikia/Blogs/BlogLockdown.php

```diff
@@ -152,4 +152,23 @@ public static function userCan( $title, $user, $action, &$result ) {
 
 		return $return;
 	}
+
+	/**
+	 * Checks permission before delete action on blog articles
+	 *
+	 * @param Article $article
+	 * @param Title $title
+	 * @param User $user
+	 * @param array $permission_errors
+	 *
+	 * @return boolean because it's a hook
+	 */
+	public static function onBeforeDeletePermissionErrors( &$article, &$title, &$user, &$permission_errors ) {
+		// Only users with delete permission can delete a blog article
+		if ( $title->getNamespace() == NS_BLOG_ARTICLE && !$user->isAllowed( 'delete' ) ) {
```

Comment:
`delete` seems like a really general permission name. Will this get us into trouble if the user is allowed to delete other things like regular wiki pages or article comments? Do you know what delete specifically refers to?